app/views/layouts/application.html.erb

```diff
@@ -30,13 +33,27 @@
       window.SCP.userAccessToken = '<%= get_user_access_token(current_user) %>';
       window.SCP.environment = '<%= Rails.env %>';
 
+      window.SCP.hasLoggedPageView = false;
+
+      <% if controller_name == 'site' && action_name == 'covid19' %>
+      window.SCP.featuredSpace = 'covid19'
+      <% end %>
```

Comment:
this probably has to be here for loading timing reasons, but if possible, it would be great to relocate it to a script on `covid19.html.erb`, just so as much covid-specific stuff is kept in one place as possible

Reply:
Worth trying.  I just did, and see the current location is indeed required for timing reasons.

We manually log `page:view` to Mixpanel, and without this variable available before our React code loads, the `featuredSpace: covid19` event property is omitted in Mixpanel log posts.  So the current location stays for now.